src/sinks/influxdb/logs.rs

```diff
@@ -686,12 +711,12 @@ mod integration_tests {
     async fn influxdb2_logs_put_data() {
         onboarding_v2().await;
 
-        let ns = format!("ns-{}", Utc::now().timestamp_nanos());
+        let measure = format!("vector-{}", Utc::now().timestamp_nanos());
 
         let cx = SinkContext::new_test();
 
         let config = InfluxDbLogsConfig {
-            namespace: ns.clone(),
+            measurement: Some(measure.clone()),
```

Comment:
CI is reporting a build failure here on a missing `namespace` in the initializer.

Reply:
Thanks for spotting this, added `namespace` now.